src/snowflake/snowpark/session.py

```diff
@@ -654,6 +657,14 @@ def __init__(
                 _PYTHON_SNOWPARK_ENABLE_QUERY_COMPILATION_STAGE, False
             )
         )
+        self._generate_multiline_queries: bool = (
+            self._conn._get_client_side_session_parameter(
+                _PYTHON_SNOWPARK_GENERATE_MULTILINE_QUERIES, True
+            )
+        )
+        if self._generate_multiline_queries:
+            self._enable_multiline__queries()
+
```

Comment:
tiny nit: can we name it `_enable_multiline_queries()` note the double underscore in current name.